_posts/2016-01-13-OSS-Expectations.markdown

```diff
@@ -0,0 +1,89 @@
+
+Beautiful. This is exactly how I feel, and in over a year these expectations have been met. Some of our libraries have become big, and our apps have received small feature PRs. We're pleasantly surprised when it happens, but we don't expect it.
+
+### What didn't we expect?
+
+I didn't expect to be told face to face how many people have read, and learned from our codebases. We get around 120 unique clones of our iOS apps every week. People tell us that it's where they found a certain technique, or that they could see how the trade-offs were made for certain decisions.
+
+I also under-estimated how useful open code is in encouraging a culture of writing. Ash once quoted [me in a tweet](https://twitter.com/ashfurrow/status/676814159363842048) saying:
+
+> Order of importance: blog posts > types > tests
+
+While it is a joke against functional programmers, in general; having these huge codebases gives a lot to talk about. A commit says the what, a pull request the how and a blog post gives the why. Writing about our code allows us to provide documentation for future employees with the context around how decisions were made. Nobody _wants_ to ship messy code, but a lot of the time you choose to in order to provide something positive.
+
+### How does Open Source affect security
+
+Artsy has a [security policy](https://www.artsy.net/security), with bounties and ways to report issues. From our perspective so far, having this open has not affected the reports we receive. If 1Password [can say](https://teams.1password.com/white-paper/1Password%20for%20Teams%20White%20Paper.pdf)
+
+> We believe that openness always trumps “security through obscurity”.
+
+Then show off their algorithms that keep passwords safe, then so long as we're careful about our keys and stick to best security practices. We seem to be doing alright.
+
+
+### What is a good approach to talk to the legal team?
+
+I have a friend who works in a very large company. He told me that he was aiming to Open Source his app. He had had a few meetings with the legal team that didn't really go anywhere. After a few meetings, he explained that they weren't aiming for community-building with this, but aiming to develop in the open. This change of perspective, changed the tone of the meetings from then on in. It became much easier to start the political work necessary to even begin working on the technical aspect.
+
+### Alright, call to action to wrap up.
+
+* Working in the open is a great way to raise the profiles of your team, and the individuals on it.
+* It can make it easier to hire, because you can "[show](https://github.com/artsy/mobile/)", not "[tell](https://www.artsy.net/article/artsy-jobs-mobile-engineer)" with respect to your team culture.
```

Comment:
Ah ok, if this is all of what you meant by the hiring benefit earlier, than ignore that earlier comment, or maybe refer to it.